Gopkg.lock

```diff
@@ -2,46 +2,81 @@
 
 
 [[projects]]
+  digest = "1:180876db3ec295bb9f0babec5ca926fe9f2036b747b7c5bfcd13b333023e7cfd"
```

Comment:
Do you also need to include the Gopkg.toml?

Reply:
Didn't get any changes in that file after running dep -- ensure